include/NovelRT.Interop/Input/NrtBasicInteractionRect.h

```diff
@@ -23,7 +23,6 @@ extern "C"
     NrtResult Nrt_Input_BasicInteractionRect_setLayer(NrtBasicInteractionRect object, int32_t value);
     int32_t Nrt_Input_BasicInteractionRect_getActive(NrtBasicInteractionRect object);
     NrtResult Nrt_Input_BasicInteractionRect_setActive(NrtBasicInteractionRect object, int32_t value);
-    NrtResult Nrt_Input_BasicInteractionRect_executeObjectBehaviour(NrtBasicInteractionRect object);
```

Comment:
There were a couple duplicate methods in NovelRT.Interop (here and below in `NrtStepTimer.h`). I'd like confirmation from @capnkenny that this were proper dupes and not something that was accidentally misnamed.

Reply:
These were dupes, my bad 😅